spdx-utils/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -138,5 +138,11 @@ data class SpdxDocument(
         require(dataLicense.isNotBlank()) { "The data license must not be blank." }
 
         require(documentNamespace.isNotBlank()) { "The document namespace must not be blank." }
+
+        require((relationships.isNotEmpty() && relationships.any { it ->
+            it.relationshipType == SpdxRelationship.Type.DESCRIBES }) || documentDescribes.isNotEmpty()
+        ) {
+            "Document must have at least one relationship of type DESCRIBES or a documentDescribes must not be blank."
```

Comment:
"The document must either have at least one relationship of type 'DESCRIBES' or contain the 'documentDescribes' field."